src/mixins/java/org/spongepowered/common/mixin/core/world/entity/EntityMixin.java

```diff
@@ -204,6 +204,8 @@
     @Shadow protected abstract boolean shadow$fireImmune();
     // @formatter:on
 
+    @Shadow public abstract boolean isPassenger();
```

Comment:
This should be put in the `// @formatter` block above, and prefixed with `shadow$`, so 

```
@Shadow public abstract boolean shadow$isPassenger();
```